packages/graphql/src/fragments/primitives.ts

```diff
@@ -401,6 +402,25 @@ const UsernameNamespaceStatsFragment = graphql(
 );
 export type UsernameNamespaceStats = FragmentOf<typeof UsernameNamespaceStatsFragment>;
 
+const UsernameNamespaceMetadataStandardFragment = graphql(
+  `fragment UsernameNamespaceMetadataStandard on UsernameNamespaceMetadataStandard {
+    __typename
+    bannerImage
+    collaborators
+    description
+    externalLink
+    featuredImage
+    image
+    name
+    schema
+    signature
```

Comment:
this is a mistake on the API side, can you please omit it from the fragment and raise a low priority API ticket for Zannis?